src/pages/UserListing/UsersListingPage.tsx

```diff
@@ -0,0 +1,11 @@
+import { UsersListingPageWrapper } from './UsersListingPage.styled';
+import { UsersTable } from './components/Table/UsersTable';
+import users from '../../data/sampleUsers.json';
+
+export const UsersListingPage = () => {
+    return (
+        <UsersListingPageWrapper>
```

Comment:
Can you try this:
<UsersListingPageWrapper component={'section'}>

Reply:
done